scikeras/wrappers.py

```diff
@@ -453,7 +458,7 @@ def _pre_process_X(X):
         extra_args = dict()
         return X, extra_args
 
-    def fit(self, X, y, sample_weight=None, **kwargs):
+    def fit(self, X, y, sample_weight=None, warm_start=True, **kwargs):
```

Comment:
This should be `warm_start=False`. This is what was causing the `OutputShape` tests to fail.

Reply:
D'oh. Thanks for the catch.